### PR TITLE
Updating user agent for all cli requests

### LINKED ISSUE #3: Set client user agent
As we track API usage internally, it would be useful to know how often our CLI tool is being used. We should overwrite the Guzzle user agent to `SpinupWP/VERSION`. `VERSION` is the current CLI tool version.

### PR DESCRIPTION
**Pull request by katelynnbarlowe**
Resolves #3

### CODE REVIEW

File: app/Commands/BaseCommand.php [hunk 1]
```diff
@@ -34,23 +34,24 @@ public function handle(): int
...
+                    'Authorization' => "Bearer {$this->apiToken()}",
+                    'Accept'        => 'application/json',
+                    'Content-Type'  => 'application/json',
+                    'User-Agent'    => 'SpinupWP/' . config('app.version'),
+                ],
+            ];
             if (!$this->spinupwp->hasApiKey()) {
-                $this->spinupwp->setApiKey($this->apiToken())->setClient();
+                $this->spinupwp->setApiKey($this->apiToken());
             }
```
Comment (A5hleyRich) [inline app/Commands/BaseCommand.php:49]: Is this needed now that we're always setting the client?

File: app/Commands/BaseCommand.php [hunk 2]
```diff
@@ -34,23 +34,24 @@ public function handle(): int
...
-                        'http_errors' => false,
-                        'headers'     => [
-                            'Authorization' => "Bearer {$this->config->get('api_token', $this->profile())}",
-                            'Accept'        => 'application/json',
-                            'Content-Type'  => 'application/json',
-                        ],
-                    ])
-                );
+                $clientOptions['base_uri'] = $this->config->get('api_url', $this->profile());
             }
```
Comment (A5hleyRich) [inline app/Commands/BaseCommand.php:53]: This conditional could be cleaned up a bit if the config helper had a default value property. Then you could just inline the base URL into the `$clientOptions` array. The `config` helper in Laravel does this: https://laravel.com/docs/8.x/helpers#method-config

**Review by A5hleyRich — CHANGES_REQUESTED** [review]

File: app/Commands/BaseCommand.php (hunk 1 above)
Comment (katelynnbarlowe) [reply app/Commands/BaseCommand.php:49]: I suppose not if the SDK only uses the api key to setup the client. I wasn't sure, but that seems to be the case.

**Review by A5hleyRich — CHANGES_REQUESTED** [review]
@katelynnbarlowe, remember to check the tests!

**Comment by katelynnbarlowe** [comment]
@A5hleyRich yeah I was working on them this evening when I had to stop. It's not ready! 😄

**Comment by A5hleyRich** [comment]
That's my bad https://media.giphy.com/media/jUwpNzg9IcyrK/giphy.gif

**Comment by A5hleyRich** [comment]
💯

**Pull request merged**